#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_use_trim_to_avoid_signature_problems.dpatch by David Gil <dgil@telefonica.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Added trim() to GetSingleSignatureReference in order to avoid 
## DP: problems when signatures contain spaces (this happens with snortcenter)

@DPATCH@
diff -urNad acidbase~/includes/base_signature.inc.php acidbase/includes/base_signature.inc.php
--- acidbase~/includes/base_signature.inc.php	2009-11-27 14:10:00.000000000 -0500
+++ acidbase/includes/base_signature.inc.php	2009-11-27 14:13:48.000000000 -0500
@@ -108,7 +108,7 @@
 
 
 
-   $tmp_ref_system_name = strtolower($ref_system);
+   $tmp_ref_system_name = trim(strtolower($ref_system));
    if ( in_array($tmp_ref_system_name, array_keys($GLOBALS['external_sig_link'])) )
    {
       if ($tmp_ref_system_name == "local_rules_dir")
